internal/ast/utilities.go

```diff
@@ -1029,6 +1029,26 @@ func IsStatic(node *Node) bool {
 	return IsClassElement(node) && HasStaticModifier(node) || IsClassStaticBlockDeclaration(node)
 }
 
+func CanHaveSymbol(node *Node) bool {
+	switch node.Kind {
```

Comment:
All the JSDoc tags should be dropped from here, replaced with JSExportAssignment JSTypeAliasDeclaration, CommonJSExport (I think).
Also I'm pretty sure Identifier is JS-only and unsupported in Corsa, so can be dropped as well.